Section 3.md

```diff
@@ -99,6 +105,13 @@ While it does not yet possess technologies like Gravitational Singularity reacto
 
 While usually the larger corporations devise their own power grid and manage their own planets, moons, orstations, FPE is often consulted and hired when a less than usual approach needs to be taken, and their products are on high demand.
 
+- **Morpheus Cyberkinetics**
+
+Originally a small company focused on research of the interface of the human mind to cybernetic parts, founded just prior to the explosion in robotics technology. This company produced the first of the truly high-end cybernetics, interfacing directly with the human nervous system, even giving a very rudimentary sense of touch. There are some rumors that Zeng-Hu had seized their results through corporate espionage, quickly producing their own line of prosthetics that eclipsed Morpheus� own.
```

Comment:
Founded just prior to the explosion in robotics technology, Morpheus was originally a smaller company focused on research of the interface designed for human mind to cybernetic connection. Eventually, Morpheus came to produce the first high-end cybernetic tech, interfacting directly with human nervous system, even giving a rudimentary sense of touch. There are rumors that Zeng-Hu had seized their results through corporate espionage, quickly producing their own line, but is unclear if those rumors were true.